apps/store/themes/store/helpers/navigation.js

```diff
@@ -24,10 +24,8 @@ var format = function(context, data, page, area, meta) {
 
 var resources = function(page, meta) {
 	return {
-
-		js : [  'asset-helpers.js', 'navigation.js', 'popover.js', 'jquery.validate.js', 'jquery.cookie.js', 'search.js', 'list_assets.js', 'categorization.js' , 'refresh-categorization.js', 'taxonomy_view.js','wso2-visual-elements.js'],
-		css : [ 'navigation.css','style-taxonomy.css', 'categorization.css', 'theme-categorization.min.css']
-
+		js : ['asset-helpers.js', 'navigation.js', 'popover.js', 'jquery.validate.js', 'jquery.cookie.js', 'search.js', 'list_assets.js', 'categorization.js', 'categorization-data.js', 'taxonomy_view.js','wso2-visual-elements.js'],
+		css : ['navigation.css','style-taxonomy.css', 'categorization.css', 'theme-categorization.min.css']
```

Comment:
@thushara35 Thanks for pointing out. This issue has been already fixed by 6683a3b78b27d06802510cc8a7ecf94575b93d49